weather_mv/loader_pipeline/regrid.py

```diff
@@ -96,6 +98,11 @@ def apply(self, uri: str):
 
             src.flush()
 
+            # Clear the metview temporary directory.
```

Comment:
Why not use tempdir's context manager and just make a directory that goes away from hen you exit the context?

Reply:
We've tried digging deep into MetView's sources, and they don't offer an interface for users to control how their tempfiles get cleaned up (or, where the tempfile is). Instead, they've more-or-less "hard-coded" a pattern where they'll clean up caches once the process exits. That doesn't suit our needs, since the dataflow instance will stay on for a long time.